README.md

```diff
@@ -60,6 +65,22 @@ In this case you can define a regular expression, which matches all user agent s
 ```
 This regular expression would match any user agent string, that ends with *My Example Webapp*.
 
+### Add a separate links specific to some device/platform
+
+There are cases when there are separate applications meant for phone and tablet of the same OS. For this purpose you can use the OS related metatag with platform-device suffix, e.g.
+
+#### iOS - iPhone app link can be set with the following metatag:
+```html
+  <meta name="smartbanner:button-url-apple-iphone" content="https://itunes.apple.com/iphonelink">
+```
+
+#### iOS - iPad app link can be set with the following metatag:
+```html
+    <meta name="smartbanner:button-url-apple-ipad" content="https://itunes.apple.com/ipadlink">
+```
+
+**Note:** Is is important that if 'global' links, like `<meta name="smartbanner:button-url-apple"/>` or `<meta name="smartbanner:button-url-google"/>` will be set, device related app links will be omitted (-phone; -tablet).
```

Comment:
I did test your code on iPhone and iPad and I did not succeed. Link turned out to be `http://127.0.0.1:8080/undefined` when running the local server (`npm run server`) of your PR code.

Also, considering how we're dealing with hiding the banner on some User Agents, I'd call for not requiring the omission of global link, but using it as fallback if none of the more specific definitions are satisfactory, e.g. on iPod.